### DISCUSSION TITLE
Strange behaviour MQTT

### DISCUSSION THREAD

**Discussion by jdrescher2006** [syssi/esphome-jbd-bms#75] [General]
Hi, I have a problem with the MQTT messages. 
So the normal MQTT message name is "jbd-bms-uart".
But sometimes there are other message names, like for example "P_VS1_5jbd-bms-uart". This happens quite often so that my MQTT broker gets spammed with lots of these strange message names.

![fehler_mqtt_jbd-bms-uart](https://github.com/syssi/esphome-jbd-bms/assets/12234756/06f61cb0-6cbd-4ced-a62a-b066bbc80577)

**Comment by syssi** [comment]
Please recompile your ESPHome node using the latest release. It looks like there was something buggy in the past. If this doesn't help try to replace the ESP module.
